playground/src/App.tsx

```diff
@@ -96,26 +174,34 @@ Cycles = <available with Miden VM v0.4>`);
   * It runs the Rust program that is imported above.
   */ 
 
-    const proveProgram = async () => {
-      init().then(() => {
-        try {
-          const resp = prove_program(code, inputs, numOfOutputs);
-          console.log(resp.toString());
-          const stack_output = resp.slice(0, numOfOutputs);
-          const overflow_addrs = resp.slice(numOfOutputs, resp.length);
-          setOutput(`Miden VM Program Output
+  const proveProgram = async () => {
+    init().then(() => {
+      try {
+        if (!tryParseJSONObject(inputs)) {
+          setOutput(`Error: Invalid JSON input`);
+          return;
+        }
+        if (!checkStackInitAndAdviceTape(inputs)) {
+          setOutput(`Error: Invalid inputs - no stack_init or advice_tape`);
+          return;
+        }
```

Comment:
This code is duplicated above. Let's make a helper

Reply:
better?